### ISSUE TITLE
MS 156: George Russell aerial photographs

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/dams_project_mgmt#333]
## Section 1: Archival Assessment/Collection Background

Information about this collection and its workflow can be found on the existing project plan: https://docs.google.com/document/d/1uSxyB2mUtgejqRZUO-EftH8AtsKnq7Ekw_-wa9Uow1M/edit

_The following sections have not been completed:_

## Section 5: DAMS Development Assessment

Development work required?: Y/N   (if Y →)
    
Brief description of new feature:
    

Subject Matter Expert(s):
    
Developer(s):

## Section 6: Display Assessment

Modifcations necessary for collection landing page:

Thumbnail Image for DAMS home page:

Brief Description:

Customizations, e.g., new format icons:

## Section 7: Metadata & File Locations

Metadata spreadsheet:
Files:

## Actions & Acceptance Criteria 
_(workflow specific to each collection, i.e., the steps required to prepare it for ingest)_

- [ ]  Collection Information and planning/assessment sections (1-7) completed
- [ ]  Initial project meeting
- [ ]  Metadata spreadsheet prepared
- [ ]  Files assessed and prepared for ingest
- [ ]  Customizations for display are discussed and documented

**Comment by rmjaffe** [comment]
@tmariamora @chesleyperry Do we know the status on this?  I'm not certain who is currently in possession of this project -- DI or SC&A? Or if there is no activity and none is expected for the time being, should I put the project card in Planning?

**Comment by tmariamora** [comment]
Sorry... this was buried. Basically, in light of Anne's retirement, I have told Kate to hold off on this project until we have staff to support. So my vote would be to move the card. @chesleyperry please speak up if you think we should be moving forward.

**Comment by chesleyperry** [comment]
Yes - I think planning is best.  Beyond Ann's retirement, we need access to the physical collection for processing and digitization.  It's not a high priority to support instruction, so keeping it on hold in planning seems appropriate.
